students/marc_charbo/lesson05/mailroom_v3.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+import logging
+import logging.config
+
+#logging.cong file
+logging.config.fileConfig('logging.conf')
+# create logger
+logger = logging.getLogger('mailRoom')
+
+DONOR_LIST = {'Jim':[25.00,150.00,2000.00,100000.00],'Linda':[10000.25],'Bob':[5.03,100.01,6.00]}
+
+def send_letter():
+    """write thank you note to all users in donor list"""
+    for key, value in DONOR_LIST.items():
+        file = open(key+'.txt', 'w')
+        file.write('Dear {},\n'.format(key))
+        file.write('\n')
+        file.write('\tThank you for your very kind donation of ${:.2f},\n'.format(sum(value)))
+        file.write('\n')
+        file.write('\tIt will be put to very good use.\n')
+        file.write('\n')
+        file.write('\t\t\tSincerely,\n')
+        file.write('\t\t\t\t-The Team')
```

Comment:
better to define all this as a single string (using triple quotes so you don't need all the `\n` and `\t` )and then write to file